pkg/cmd/opts/install.go

```diff
@@ -355,7 +355,7 @@ func (o *CommonOptions) InstallHelm() error {
 		return err
 	}
 
-	clientURL := fmt.Sprintf("https://storage.googleapis.com/kubernetes-helm/helm-v%s-%s-%s.tar.gz", stableVersion, runtime.GOOS, runtime.GOARCH)
+	clientURL := fmt.Sprintf("https://get.helm.sh/helm-v%s-%s-%s.tar.gz", stableVersion, runtime.GOOS, runtime.GOARCH)
```

Comment:
There is a second URL above in the `InstallTiller` command which also needs to be updated. It is probably not very relevant but it is good to be consistent.

Reply:
Submitting a new PR